.github/workflows/issue-triage.yml

```
@@ -5,9 +5,12 @@ on:
     types: [opened]
 
 permissions:
-  contents: read
+  contents: write
   issues: write
+  pull-requests: write
   id-token: write
+  workflows: write
```

actionlint (1.7.10)

unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)